README.md

```diff
@@ -1104,6 +1105,48 @@ bash -v scriptname
 bash -x scriptname
 ```
 
+# 5. Package Management
+There are two types of packages available for linux systems.
+* Debian Packages (.deb)
+* RedHat Package Manager (.rpm)
+
+## 5.1 Debian Package (deb)
+deb is the format, as well as extension of the software package format for the Debian distribution and its derivatives.
+
+### 5.1 A.`Update Packages`
+The Debian package manager updates its package data by ```bash sudo apt update```
+
+### 5.1 B.`Upgrade Packages`
+After updating package information if there is any update available then run ```bash sudo apt upgrade``` to get the available update.
+
+### 5.1 C.`Install Package`
+You can install new package by ```bash sudo apt install <package-name>```
+
+### 5.1 D.`Remove/Uninstall Package`
+To remove/uninstall package ```bash sudo apt autoremove <package-name>```
+
+### 5.1 E.`Install Required Dependecies`
+Sometimes to fullfill the reqirements of package you need to install its dependencies it can be done by ```bash sudo apt-get install --fix-missing```
+
+### 5.1 F.`List All Installed Packages`
+To list all installed packages ```bash dpkg -l```
+
+## 5.2 RedHat Packages (rpm)
+RPM Package Manager(RPM) (originally Red Hat Package Manager; now a recursive acronym) is a package management system. The name RPM refers to the following: the .rpm file format, files in the .rpm file format, software packaged in such files, and the package manager program itself. RPM was intended primarily for Linux distributions; the file format is the baseline package format of the Linux Standard Base.
+
+### 5.2 A.`Install Package`
+You can install new package by ```sudo rpm -ivh <package-name>```
```

Comment:
You previously used `apt` which is a frontend to `dpkg` to show installation etc. so it would be better to also show the commonly used `rpm` frontends `yum` and the latest `dnf`